VAMobile/src/api/prescriptions/getPrescriptions.tsx

```diff
@@ -0,0 +1,100 @@
+import { useQuery } from '@tanstack/react-query'
+import { TFunction } from 'i18next'
+import { contains, filter, sortBy } from 'underscore'
+
+import {
+  PrescriptionSortOptionConstants,
+  PrescriptionsGetData,
+  PrescriptionsList,
+  RefillStatus,
+  RefillStatusConstants,
+} from 'api/types'
+import { get } from 'store/api'
+import { getTextForRefillStatus } from 'utils/prescriptions'
+
+import { prescriptionKeys } from './queryKeys'
+
+/**
+ * Fetch user prescriptions
+ */
+const getPrescriptions = async (): Promise<PrescriptionsGetData | undefined> => {
+  const params = {
+    'page[number]': '1',
+    'page[size]': '5000',
+    sort: 'refill_status', // Parameters are snake case for the back end
+  }
+  return get<PrescriptionsGetData>('/v0/health/rx/prescriptions', params)
+}
+
+/**
+ * Returns a query for user prescriptions
+ */
+export const usePrescriptions = (options?: { enabled?: boolean }) => {
+  return useQuery({
+    ...options,
+    queryKey: prescriptionKeys.prescriptions,
+    queryFn: () => getPrescriptions(),
+    meta: {
+      errorName: 'getPrescriptions: Service error',
+    },
+  })
+}
+
+export const filterAndSortPrescriptions = (
```

Comment:
Since this function isn't directly connected to the query, I don't think it should be stored here. [src/utils/prescriptions.tsx](https://github.com/department-of-veterans-affairs/va-mobile-app/blob/develop/VAMobile/src/utils/prescriptions.tsx) would probably be a better place for this

Reply:
moved it to the utils file